src/context/viewer-context.tsx

```diff
@@ -95,7 +114,7 @@ function useAuthedViewer() {
     }
 
     return clearUserMonitorInterval
-  }, [viewer])
+  }, [viewerId])
```

Comment:
`useEffect` doesn't do deep object comparison, so instead using the user's id in the dependency array.